apps/wallet/src/services/chains/ethereum-native-api.service.ts

```diff
@@ -0,0 +1,19 @@
+import { AccountIncomingTransfer, ChainApi, FetchTransfersInput } from '~/types/chain.types';
+import { isValidSha256 } from '~/utils/helper.utils';
+import { Account } from '../../generated/station/station.did';
+
+export class EthereumNativeApi implements ChainApi {
+  constructor(private readonly account: Account) {}
+
+  isValidAddress(address: string): boolean {
+    return isValidSha256(address);
+  }
+
+  async fetchBalance(): Promise<bigint> {
+    return BigInt(123n);
+  }
```

Comment:
huh? Looks like this file does not affect anything. Because the balance is fetched from rust's `BlockchainApi` and it is displayed correctly in the UI.

Reply:
we should be able to remove this actually since we don't use for ICP as well